config.cmake

```diff
@@ -100,7 +100,6 @@ HHVM_DEFINE(mongodb "-DMONGOC_COMPILATION=1")
 HHVM_DEFINE(mongodb "-DMONGOC_ENABLE_CRYPTO")
 HHVM_DEFINE(mongodb "-DMONGOC_ENABLE_SSL")
 HHVM_DEFINE(mongodb "-DMONGOC_ENABLE_SASL")
-HHVM_DEFINE(mongodb "-DMONGOC_TRACE=1")
```

Comment:
Why was this removed? Was it just to remove overhead, since you don't support the `mongodb.debug` INI setting?